hedera-mirror-importer/src/main/java/com/hedera/mirror/importer/migration/InitializeEntityBalanceMigration.java

```diff
@@ -18,13 +18,17 @@
 
 import com.google.common.base.Stopwatch;
 import com.hedera.mirror.importer.MirrorProperties;
+import com.hedera.mirror.importer.parser.balance.BalanceStreamFileListener;
+import com.hedera.mirror.importer.repository.AccountBalanceFileRepository;
+import com.hedera.mirror.importer.repository.RecordFileRepository;
 import jakarta.inject.Named;
 import org.flywaydb.core.api.MigrationVersion;
 import org.springframework.context.annotation.Lazy;
 import org.springframework.jdbc.core.JdbcOperations;
 
 @Named
-public class InitializeEntityBalanceMigration extends RepeatableMigration {
+public class InitializeEntityBalanceMigration extends TimeSensitiveBalanceMigration
+        implements BalanceStreamFileListener {
```

Comment:
```suggestion
public class InitializeEntityBalanceMigration extends TimeSensitiveBalanceMigration {
```

same for the other migration